core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -216,6 +216,16 @@
 				 XLAT_TABLE_USER_EXTRA)
 #endif /*!MAX_XLAT_TABLES*/
 
+#if (CORE_MMU_BASE_TABLE_LEVEL == 0)
+#if (MAX_XLAT_TABLES < (1 << 8))
+#define l1_idx_type uint8_t
```

Comment:
How about using
``` C
typedef uint8_t l1_idx_t;
```
instead? Using `#define` instead of typedef looks like a funny way of avoiding a checkpatch warning.

Reply:
agree. If the typedef check is irrelevant for this project, then it can be disabled in `.checkpatch.conf` in the root of the project.

Reply:
@jenswi-linaro agreed.
@Izikd we could disable the check easily, yes, but I think I'd rather keep it. Most of the time a new typedef is not what we want and this case is an exception.